resolwe_bio/processes/workflows/cutadapt_star_fc_quant_tf.yml

```diff
@@ -0,0 +1,171 @@
+- slug: workflow-cutadapt-star-fc-quant-single
+  name: Cutadapt - STAR - FeatureCounts (3' mRNA-Seq, single-end)
+  data_name: "{{ reads|sample_name|default('?') }}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:quant:featurecounts:single
+  category: Pipeline
+  description: |
+    This 3' mRNA-Seq pipeline is comprised of QC, preprocessing,
+    alignment and quantification steps.
+
+    Reads are preprocessed by __Cutadapt__ which removes adapters, trims
+    reads for quality from the 3'-end, and discards reads that are too short
+    after trimming. Preprocessed reads are aligned by __STAR__
+    aligner. For read-count quantification, the __FeatureCounts__ tool
+    is used.
+
+    QC steps include downsampling, QoRTs QC analysis and alignment of
+    input reads to the rRNA/globin reference sequences. The reported
+    alignment rate is used to asses the rRNA/globin sequence depletion
+    rate.
+  input:
+    - name: reads
+      label: Select sample(s)
+      type: data:reads:fastq:single
+    - name: star_index
+      label: Genome
+      type: data:genomeindex:star
+      description: |
+        Genome index prepared by STAR aligner indexing tool.
+    - name: annotation
+      label: Annotation
+      type: data:annotation
+      description: |
+        Genome annotation file (GTF).
+    - name: rrna_reference
+      label: Indexed rRNA reference sequence
+      type: data:genomeindex:star
+      description: |
+        Reference sequence index prepared by STAR aligner indexing tool.
+    - name: globin_reference
+      label: Indexed Globin reference sequence
+      type: data:genomeindex:star
+      description: |
+        Reference sequence index prepared by STAR aligner indexing tool.
+    - name: show_advanced
+      label: Show advanced parameters
+      type: basic:boolean
+      default: false
+    - name: downsampling
```

Comment:
Listing downsampling options before cutadapt options is misleading. It may lead the user into thinking that filtering and trimming is done on downsampled FASTQ files.